easybuild/easyconfigs/p/Python/Python-3.6.6-foss-2018b.eb

```diff
@@ -0,0 +1,223 @@
+name = 'Python'
+version = '3.6.6'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['7d56dadf6c7d92a238702389e80cfe66fbfae73e584189ed6f89c75bbf3eda58']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('libreadline', '7.0'),
+    ('ncurses', '6.1'),
+    ('SQLite', '3.24.0'),
+    ('XZ', '5.2.4'),
+    ('GMP', '6.1.2'),  # required for pycrypto
+    ('libffi', '3.2.1'),  # required for cryptography
+    # OS dependency should be preferred if the OS version is more recent then this version,
+    # it's nice to have an up to date OpenSSL for security reasons;
+    # Python 3.7.0 requires OpenSSL >= 1.0.2, uncomment line below if OS version is too old (e.g. CentOS 6)
+    # ('OpenSSL', '1.1.0h'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+exts_download_dep_fail = True
+
+# workaround for "undefined symbol: __stack_chk_guard"
+# see also https://software.intel.com/en-us/forums/intel-c-compiler/topic/610514
+buildopts = 'LDFLAGS="$LDFLAGS -lssp"'
```

Comment:
Is this really needed in the foss version or is it just convenient to keep the identical?

Reply:
The latter (and it doesn't hurt)

Reply:
btw, where does ssp come from? Python itself?

Reply:
No, `GCCcore`, see `lib64/libssp.so`